ACE/tests/Native_Condition_Variable_Test.cpp

```diff
@@ -0,0 +1,47 @@
+
+//=============================================================================
+/**
+ *  @file    Native_Condition_Variable_Test.cpp
```

Comment:
I would propose to remove Native_ from the name of this test, it just tests the condition variable behavior on any platform